Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -72,15 +88,21 @@ def __init__( self, ceUniqueID ):
     self.outputURL = 'gsiftp://localhost'
     self.gridEnv = ''
     self.proxyRenewal = 0
+    self.extraSubmitString = self.ceParameters.get('ExtraSubmitString', '').decode('string_escape')
+
+    ## see note on getCondorLogFile, why we can only use the global setting
+    self.workingDirectory = gConfig.getValue( "Resources/Computing/HTCondorCE/WorkingDirectory",
```

Comment:
Well, ideally the WMSAdministrator should not care about the CE parameters at all. It should rather use the CE object for all the CE related operations. and the CE object knows about its parameters